x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -7617,7 +7617,6 @@
     "xpack.enterpriseSearch.overview.productCard.launchButton": "{productName}の起動",
     "xpack.enterpriseSearch.overview.productCard.setupButton": "{productName} のセットアップ",
     "xpack.enterpriseSearch.overview.setupCta.description": "Elastic App Search および Workplace Search を使用して、アプリまたは社内組織に検索を追加できます。検索が簡単になるとどのような利点があるのかについては、動画をご覧ください。",
-    "xpack.enterpriseSearch.overview.setupCta.title": "あらゆる規模のチームに対応するエンタープライズ級の機能",
```

Comment:
[super optional, only if you're cool with opening up the json file] Since the copy didn't change, save the localizers a bit of time if we rename + order the ID alphabetically instead of deleting it:

```
    "xpack.enterpriseSearch.productSelectorCalloutTitle": "あらゆる規模のチームに対応するエンタープライズ級の機能",
```

Reply:
Super nice catch! Thanks for the amazing review Constance!